docker/install_airflow.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+# These are picked up from the Docker ENV.
+AIRFLOW_DIR=${INSTALL_DIR}/${PARCEL_NAME}
+#PYVER=$(echo "$PYTHON_VERSION" | awk -F. '{print $1"."$2}')
+#PYMAJVER=$(echo "$PYTHON_VERSION" | awk -F. '{print $1}')
+
+PATH="${AIRFLOW_DIR}/bin:${PATH}"
+PIPOPTS=""
+#export SLUGIFY_USES_TEXT_UNIDECODE=no
+export AIRFLOW_GPL_UNIDECODE=yes
+
+echo "*** Installing Airflow..."
+pip $PIPOPTS install apache-airflow=="${AIRFLOW_VERSION}"
+
+echo "*** Installing Airflow plugins..."
+echo "** Installing Airflow[celery]."
```

Comment:
We can confirm, but I'm pretty sure you should also need to include the version of Airflow you want these subpackages to be installed as:
ex:
pip $PIPOPTS install "apache-airflow[celery]==${AIRFLOW_VERSION}"

Reply:
I will add versioning in the next set of PRs.
There is a much bigger issue of pinning *all* of the dependencies that has to be solved in some manner.  Otherwise, tomorrow's rebuild with the same exact build code may result in newer deps getting installed.